attributes/default.rb

```diff
@@ -1,6 +1,7 @@
 default['packer']['url_base'] = 'https://dl.bintray.com/mitchellh/packer'
 default['packer']['version'] = '0.8.6'
 default['packer']['arch'] = kernel['machine'] =~ /x86_64/ ? 'amd64' : '386'
+default['packer']['zipfile'] = ''
```

Comment:
I am concerned with an order-of-operations where there could be a potential that chef doesn't see this attribute being initialized, but it's probably not necessary. This is a habit of mine. I think it does two things:
- Keeps the definition of all the cookbook's attributes in a single location; someone reading the cookbook can pick out all of its configurable attributes without weeding through code in the recipe.
- very minor chance, but possibly averting a case where the node attribute isn't initialized where we think it should.